frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/catering-lines-feature-manager.ts

```diff
@@ -12,6 +12,7 @@ import type { CateringLine } from 'src/app/shared/types/catering-line';
 import type { AppState } from 'src/app/state/app.state';
 import { selectVisibleCateringLines } from 'src/app/state/application/selectors/shared.selectors';
 import type OlMap from 'ol/Map';
+import type { Element as Elem } from 'digital-fuesim-manv-shared';
```

Comment:
In my opinion, it would be okay to shadow the global `Element` with this import, since we're not interacting with the DOM from the feature managers and not obvious what `Elem` should be when you see it in the code.

If you change it, please do so in the other feature managers, too.

Reply:
This could lead to Problems in the Feature Managers since they know the Type element as the specific Element that they manage

Reply:
I would suggest renaming the type parameter `Element` to something like `ManagedElement` to differentiate between types and type parameters.

Reply:
I renamed both